simulariumio/filters/translate_filter.py

```diff
@@ -67,28 +66,5 @@ def apply(self, data: TrajectoryData) -> TrajectoryData:
                 else:
                     translation = self.default_translation
                 # apply translation
-                display_type = data.agent_data.display_type_for_agent(
-                    time_index, agent_index
-                )
-                translate_subpoints = (
-                    max_subpoints > 0 and display_type != DISPLAY_TYPE.SPHERE_GROUP
-                )
-                if translate_subpoints:
-                    sp_items = self.get_items_from_subpoints(
-                        data.agent_data, time_index, agent_index
-                    )
-                    if sp_items is None:
-                        translate_subpoints = False
-                    else:
-                        # translate subpoints for fibers
-                        n_items = sp_items.shape[0]
-                        for item_index in range(n_items):
-                            sp_items[item_index][:VALUES_PER_3D_POINT] += translation
-                        n_sp = int(data.agent_data.n_subpoints[time_index][agent_index])
-                        data.agent_data.subpoints[time_index][agent_index][
-                            :n_sp
-                        ] = sp_items.reshape(n_sp)
-                if not translate_subpoints:
-                    # translate agent position for non-fibers
-                    data.agent_data.positions[time_index][agent_index] += translation
```

Comment:
Checking my understanding: Dan's updates to the viewer make this redundant because we add translation and rotation in the shader now

Reply:
So prior to Dan's changes, the agent position data and rotation data would be ignored for fibers, the viewer would only pay attention to the subpoint position data. With Dan's changes, the position data and rotation data are now taken into account in the viewer, but if the .simularium trajectory has the rotation and position data zeroed out, then we don't even tap into the work Dan did. This change set will allow future trajectories converted with simulariumio to take advantage of these changes Dan made in the viewer!